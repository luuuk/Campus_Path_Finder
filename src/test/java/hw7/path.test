# This test is for graphs with a relatively long path

LoadGraph test test.tsv

ListChildren test a

FindPath test a f