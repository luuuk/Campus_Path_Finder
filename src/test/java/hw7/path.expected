# This test is for graphs with a relatively long path

loaded graph test

the children of a in test are: b(0.500) c(0.333) e(1.000)

path from a to f:
a to c with weight 0.333
c to f with weight 1.000
total cost: 1.333